requirements-dev.txt

```diff
@@ -1,2 +1,7 @@
 coverage
 pytest
+mypy
+types-requests
```

Comment:
these typing dependencies were copied from [census](https://github.com/chanzuckerberg/cellxgene-census/blob/main/.pre-commit-config.yaml), and were all needed to get various types of errors to go away